.releaserc.yml

```
@@ -38,6 +38,18 @@ plugins:
     }]
   - "@semantic-release/exec"
 
+  # Plugin to automatically backmerge main → develop after a release,
+  # keeping develop in sync with hotfixes published from main.
+  - ["@saithodev/semantic-release-backmerge", {
+      backmergeBranches: [
+        { from: "main", to: "develop" }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)